test/stat.jl

```diff
@@ -0,0 +1,91 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+using Base.Filesystem: StatStruct, samefile
+
+show(DevNull, StatStruct())
+mktempdir() do d
+    mktemp(d) do f, io
+        @test typeof(stat(RawFD(fd(io)))) <: StatStruct
+        @test typeof(stat(fd(io))) <: StatStruct
+        @test typeof(stat(splitdir(f)...)) <:StatStruct
```

Comment:
probably a pygments bug in our syntax definition where it's parsing that as `< (:StatStruct)` instead of `<: (StatStruct)`